chroot-bin/croutonxinitrc-wrapper

```diff
@@ -126,6 +126,15 @@ if [ "$xmethodtype" != "xiwi" ]; then
     fi
 fi
 
+# Apply the Chromebook keyboard map in xiwi if using Freon. 
+if [ "$xmethodtype" = 'xiwi' ] && [ ! -f "/sys/class/tty/tty0/active" ]; then
```

Comment:
Actually, the `setkbmap -model chromebook` in the above block could be factored out if you move this code up above.  Then the condition for setting the map is "xiwi or freon" (and the map exists).